script/app.js

```diff
@@ -0,0 +1,23 @@
+let addToDoButton = document.querySelector(".task__button-add");
+let toDoContainer = document.querySelector(".todo__container");
+let inputField = document.querySelector(".task__input");
+let createTask = document.querySelector(".button_create");
+let deleteTask = document.querySelector(".task__button-delete");
+let taskCard = document.querySelector(".task__card");
+
+createTask.addEventListener("click", function () {
+    if (!taskCard.style.display || taskCard.style.display === "none") {
+        taskCard.style.display = "block";
+    }
+});
+
+deleteTask.addEventListener("click", function () {
+    taskCard.style.display = "none";
```

Comment:
This means we are never deleting the task actually? Just hiding using CSS?

Reply:
yes, if i delete  then i can't create any new task again on that list . it's just a container for making todo list. delete functionality will be implemented within the todo list . Following the design i just named it delete task .